python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py

```diff
@@ -61,6 +58,24 @@ def is_paddle_func(func):
     return m is not None and m.__name__.startswith("paddle")
 
 
+def is_unsupported(func):
+    # TODO(liym27): A better way to do this.
+    if any(func in m.__dict__.values()
+           for m in (collections, pdb, copy, inspect, re, six, numpy)):
```

Comment:
Is it better to define a inner constant variable, such as `BUILTIN_LIKELY_MODULES` to deal this?

Reply:
Done